docs/conf.py

```diff
@@ -82,7 +82,7 @@
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [(master_doc, 'flakeheaven', 'FlakeHeaven Documentation', [author], 1)]
+man_pages = [(main_doc, 'flakeheaven', 'FlakeHeaven Documentation', [author], 1)]
```

Comment:
```suggestion
man_pages = [(root_doc, 'flakeheaven', 'FlakeHeaven Documentation', [author], 1)]
```

same, https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-master_doc